src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -298,8 +298,8 @@ public void del() {
     assertTrue(client.create("ts-del", 10000L));
     assertTrue(client.del("ts-del", 0, 1));
 
-    assertEquals(1000L, client.add("ts-del", 1000L, 1.1, 10000, null));
-    assertEquals(2000L, client.add("ts-del", 2000L, 0.9, null));
+    assertEquals(1000L, client.add("ts-del", 1000L, 1.1, 10000));
```

Comment:
Why did you change these tests?

Reply:
@gkorland I just added these in #60 mostly copying from `testAdd()` test. Due to change in `add(...)` method, now we have to type-cast the null. Instead of doing that, I used simplified versions because actual `add(...)` is being (and should be) tested in `testAdd()` and we should be more concerned about testing `del(...)` command here, not `add(...)`.